DESCRIPTION

```diff
@@ -0,0 +1,55 @@
+Type: Package
+Package: GAMBLR.viz
+Title: Make Plots for Genomic Analysis of Mature B-cell Lymphomas in R
+Version: 0.1.0
+Authors@R:
+    person("Vladimir", "Souza", , "vsouza@bcgsc.ca", role = c("aut", "cre"))
```

Comment:
I also think, at some point, the authors section should be updated to match what' is specified in the parent package:
```
Authors@R: c(
    person("Ryan", "Morin", , "rdmorin@sfu.ca", role = c("aut", "cre"),
           comment = c(ORCID = "0000-0003-2932-7800")),
    person("Kostia", "Dreval", role = "aut",
            comment = c(ORCID = "0000-0002-6214-2843")),
    person("Laura", "Hilton", role = "ctb",
           comment = c(ORCID = "0000-0002-6413-6586")),
    person("Adam", "Mattsson", , "cmattsson@bcgsc.ca", role = "aut",
           comment = c(ORCID = "0000-0002-6318-7912")),
    person("Haya", "Shaalan", role = "ctb"),
    person("Kristena", "Daley", role = "ctb")
  )
```